Frameworks/UIKit/StarboardXaml/CompositorInterface.mm

```diff
@@ -332,7 +333,12 @@ void SetParams(UIFont* font,
         _isBold = (mask & UIFontDescriptorTraitBold) > 0;
         _isItalic = (mask & UIFontDescriptorTraitItalic) > 0;
         std::wstring wideBuffer = Strings::NarrowToWide<std::wstring>(text);
-        ConstructGlyphs([[font fontName] UTF8String], wideBuffer.c_str(), wideBuffer.length());
```

Comment:
Let me try to clarify on correctness here. [UIFont familyName] for this on the reference platform would return "Franklin Gothic Medium Cond". This is the same with XAML and DWrite, as well as several websites such as http://fontsgeek.com/fonts/Franklin-Gothic-Medium-Cond-Regular which enumerate family names for fonts, suggesting that it's a property of the ttf file itself.

However, XAML/DWrite only accept "Franklin Gothic" as the _input_ family name, even though they give something else as output. This is fundamentally a mismatch on their end, so I don't know that there's anything "more correct" than this UIFont private function right now.